style.css

```diff
@@ -0,0 +1,153 @@
+body {
+  position: relative;
+  height: 100vh;
+}
+
+a {
+  text-decoration: none;
+  font-weight: normal;
+  color: black;
+}
+
+ul {
+  list-style: none;
+}
+
+.date,
+.middle-header,
+.login-btn {
+  display: none;
+}
+
+.trending-news-ul li {
+  list-style: none;
+}
+
+.news-cat-li a {
+  color: black;
+}
+
+.news-cat-li {
+  list-style: none;
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.trending-news-li {
+  color: rgb(61, 58, 58);
+}
+
+.search-li {
+  list-style: none;
+}
+
+.search-li a {
+  color: gray;
+  text-align: end;
+  font-size: 100%;
+}
+
+.header-svg svg {
+  width: 150px;
+  fill: white;
+}
+
+.middle-footer h3 {
+  font-size: 20px;
+}
+
+.middle-header {
+  position: relative;
+  width: 100%;
+}
+
+.last-footer-ul {
+  font-size: 16px;
+  padding: 0;
+  align-items: flex-start;
+}
+
+.news-cat-ul {
+  padding-left: 20px;
+}
+
+.first-header {
+  background-color: #f72210;
+  padding: 0 10px;
+  height: 45px;
+}
+
+.sub-btn,
+.login-btn {
+  height: 30px;
+  font-size: 12px;
+}
+
+.login-section {
+  gap: 20px;
+  color: rgba(255, 255, 255, 0.808);
+}
+
+footer {
+  position: absolute;
+  bottom: 0;
+}
+
+.date-stamp {
+  font-size: 13px;
+}
+
+.footer-svg svg {
+  display: block;
+  fill: #f72210;
+  width: 236px;
+  margin-bottom: 20px;
+}
+
+@media only screen and (min-width: 992px) {
+  .first-header {
+    height: 95px;
+    padding: 22px;
+  }
+
+  .header-svg svg {
+    width: 100%;
+    fill: white;
+  }
+
+  .news-category::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    content: "";
+    display: block;
+    width: 5%;
+    background-color: #f72210;
+    border-left: 12px solid white;
+    border-top: 13px solid #f72210;
+  }
+
+  ul li {
+    padding: 5px;
+    color: black;
+  }
+
+  .top-footer {
+    justify-content: space-between;
+    align-items: center;
```

Comment:
- [ ] use `justify-content-between` class instead of `justify-content: space-between;`
- [ ] use `align-items-center` class instead of `align-items: center;`